kitsune/users/tests/test_templates.py

```diff
@@ -639,6 +639,6 @@ def test_POST(self):
         eq_(w.is_active, False)
 
         self.client.post(reverse('users.edit_watch_list'), {
-            'watch_%s' % self.question.id: '1'})
```

Comment:
It was a bug in the test. The view expects the watch ID. This used to pass because we were resetting the ID sequence between tests so the question id and watch id were both 1.